state/indexer/indexer_service_test.go

```diff
@@ -139,7 +139,7 @@ func setupDB(t *testing.T) (*dockertest.Pool, error) {
 	assert.Nil(t, err)
 
 	resource, err = pool.RunWithOptions(&dockertest.RunOptions{
-		Repository: psql.DriverName,
+		Repository: "postgres",
```

Comment:
This shouldn't have been using the driver name anyway—what we want here is the _image_ name, and while that happens to coincidentally match in this case, it wasn't the right place for a dependency.